schemas/questionnaire_v1.json

```diff
@@ -55,8 +55,12 @@
       "description": "The survey (Census or Census Coverage Survey) that an address has been sampled into for participation"
     },
     "form_type": {
-      "enum": ["H", "I", "C"],
-      "description": "The form types are Household (H), Individual (I) or Communal Establishment (C)"
+      "type": "string",
+      "description": "The form types are any alphanumeric strings"
```

Comment:
This should be enforced with a `pattern`, something like `^[0-9a-z_]+$` (as underscores are also allowed).

Reply:
Added now, thanks!